doc/installation.rst

```diff
@@ -0,0 +1,38 @@
+Installation
+============
+
+**bluetooth_mesh** requires Python >= 3.5
+
+Installing/upgrading from Silvair's PyPI
+----------------------------------------
+
+To install the latest version from Silvair's PyPI at http://pypi.silvair.lan:
+
+::
+
+   $ pip install \
+      --upgrade \
+      --extra-index-url=http://pypi.silvair.lan \
+      --trusted-host pypi.silvair.lan \
+      bluetooth_mesh
+
+There are a number of extra features available, with additional dependencies:
+
+* documentation
+
+::
+
+   $ pip install bluetooth_mesh[doc]
+
+* heatmap for network diagnostic server
+
+::
+
+   $ pip install bluetooth_mesh[heatmap]
+
+* command line tools (e.g. `meshcli`)
+
+::
+
+   $ pip install bluetooth_mesh[tools]
```

Comment:
`tools` extra will be removed when we remove `platforms-clients` dependency.

Reply:
I'd like `meshcli` to still be available in this package, possibly with decreased UX when `platforms-clients` is not installed.